qunit-dom-codemod.js

```diff
@@ -120,6 +120,28 @@ export default function(file, api, options) {
     return j(`x({ count: ${count} })`).find(j.ObjectExpression).paths()[0].node;
   }
 
+  // assert.equal(find('input:checked').length, 1) -> assert.dom('input:checked').isChecked()
+  // assert.equal(find('input:checked').length, 0) -> assert.dom('input:checked').isNotChecked()
+
+  root.find(j.CallExpression, assertEqual)
+    .filter(p => isLengthWithFind(p.get('arguments').get('0')) && j.match(p.get('arguments').get('1'), {
+      type: 'Literal',
+    }))
+    .forEach(p => {
+      let findNode = p.node.arguments[0].object;
+      let count = p.node.arguments[1].value;
+      let customMessage = p.node.arguments[2];
+      let selector = findNode.arguments[0];
+
+      if (isSelector(selector) && selector.value.endsWith(':checked')) {
+        let assertion = count === 0 ? 'isNotChecked' : 'isChecked';
+
+        selector.value = selector.value.substr(0, selector.value.length - 8);
```

Comment:
Not immediately clear where the `8` is coming from. Maybe `selector.value.length - ':checked'.length` for better readability?

Or even `let [cssSelector] = selector.value.split(':checked')`?

Reply:
yeah, I did that in a follow-up commit already. good catch.